newm-server/src/main/kotlin/io/newm/server/features/earnings/model/EarningPaymentResponse.kt

```diff
@@ -0,0 +1,32 @@
+package io.newm.server.features.earnings.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class EarningPaymentResponse(
+    @SerialName("cborHex")
+    val cborHex: String,
```

Comment:
This is much simpler than we had for paying for distribution. All we need in this object is `cborHex`, `adaPrice`, and `usdAdaExchangeRate`.

Reply:
Actually, just the cborHex is fine.